modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -599,7 +599,8 @@ export default function ActionBuilder({
 								if (value === 'add-object-entry') {
 									fetchObjectDefinitions();
 								}
-								setValues({
+
+								return values.objectActionExecutorKey === value ? '' : setValues({
 									objectActionExecutorKey: value,
 									parameters: {},
 								});
```

Comment:
Hey @TaiPhamLR and @sontruongces I think we can make this code simpler and more readable by inverting the condition, like:

```ts
if (values.objectActionExecutorKey !== value) {
    return setValues({
      objectActionExecutorKey: value,
      parameters: {},
    });
}
```

Reply:
@danvb112, thank you for your feedback. The code was updated.